root/work/edit_form.tt

```diff
@@ -99,6 +99,18 @@
 
   <div class="documentation">
     [%- iswc_bubble(link_entity(work)) -%]
+
+    <div class="bubble" id="type-bubble">
+      <span id="type-bubble-default">
+        [%- l('Select any type from the list to see its description. If the work
+               doesn’t seem to match any type, just leave this blank.') -%]
+      </span>
+      [% FOREACH id IN form.field('type_id').options %]
+         <span style="display:none" class="type-bubble-description" id="type-bubble-description-[%- id.value -%]">
+           <strong>[%- add_colon(l('Description')) -%] </strong>[%- c.model('WorkType').get_description(id.value) -%]
```

Comment:
Calling `get_description` inside a loop in the template isn't the best idea (it will cause N additional database queries per page load, none of which are cached, where N = the number of work types). Calling `get_all` once in the controller as described above will fix that.

There's another problem here which is that the descriptions are not marked for translation. You'll need to create an `l_description` method in `Entity::WorkType`, similar to here: https://github.com/metabrainz/musicbrainz-server/blob/2dc039d04f49166edf20dcdd29ffe9d983cda26d/lib/MusicBrainz/Server/Entity/EventType.pm#L17-L20